src/pca-gophish-composition.service

```
@@ -13,8 +13,13 @@ ExecStartPre=/usr/bin/docker compose -f /var/pca/pca-gophish-composition/docker-
 ExecStart=/usr/bin/docker compose -f /var/pca/pca-gophish-composition/docker-compose.yml -f /var/pca/pca-gophish-composition/docker-compose.production.yml up
 # Stop composition when unit is stopped
 ExecStop=/usr/bin/docker compose -f /var/pca/pca-gophish-composition/docker-compose.yml -f /var/pca/pca-gophish-composition/docker-compose.production.yml down
-# Run gophish-init script after gophish service starts up in composition
-ExecStartPost=/bin/sh -c "cd /var/pca/pca-gophish-composition && while ! (/usr/bin/docker compose logs gophish | /bin/grep 'Starting admin server' > /dev/null); do sleep 1; /bin/echo 'Waiting for gophish startup...'; done && /usr/bin/docker compose exec -T gophish get-api-key | head -1 | /usr/bin/xargs /usr/local/bin/gophish-init"
+# Run gophish-init script after gophish service starts up in
+# composition.
+#
+# Note that the gophish-init script resides in a Python virtual
+# environment in the /var/pca/pca-gophish-composition directory:
+# https://github.com/cisagov/ansible-role-pca-gophish-composition/blob/develop/tasks/main.yml
+ExecStartPost=/bin/sh -c "cd /var/pca/pca-gophish-composition && while ! (/usr/bin/docker compose logs gophish | /bin/grep 'Starting admin server' > /dev/null); do sleep 1; /bin/echo 'Waiting for gophish startup...'; done && /usr/bin/docker compose exec -T gophish get-api-key | head -1 | /usr/bin/xargs ./.venv/bin/gophish-init"
```

I know it's unrelated to your changes but since we prefer long options I thought it worth mentioning.

```suggestion
ExecStartPost=/bin/sh -c "cd /var/pca/pca-gophish-composition && while ! (/usr/bin/docker compose logs gophish | /bin/grep 'Starting admin server' > /dev/null); do sleep 1; /bin/echo 'Waiting for gophish startup...'; done && /usr/bin/docker compose exec --no-TTY gophish get-api-key | head -1 | /usr/bin/xargs ./.venv/bin/gophish-init"
```

---

Please see commit de00fff.

---

Ah, even better!